src/components/molecules/BannerSection/BannerSection.js

```diff
@@ -21,6 +21,9 @@ const useStyles = props => {
       marginBottom: theme.spacing(props.styles.textContainer.marginBottom),
       marginRight: theme.spacing(props.styles.textContainer.marginRight),
       marginLeft: theme.spacing(props.styles.textContainer.marginLeft),
+      [theme.breakpoints.only("xs")]: {
+        marginTop: theme.spacing(props.styles.textContainer.padding - props.styles.textContainer.padding + 3),
```

Comment:
Why do we reduce the same value? 
It means `props.styles.textContainer.padding - props.styles.textContainer.padding`
If the purpose is having marginTop with a default value, I think that is better:
`marginTop: theme.spacing(3).`. 
On the other hand, when the header is setting with `position: fixed`, I wish to recommend use a default value of `10`:
`marginTop: theme.spacing(10).`. 
What do you think @scastillo-jp ?

Reply:
I added the  `marginTop: theme.spacing(10)` in the file  BannerSection.js.
     [theme.breakpoints.only("xs")]: {
        marginTop: theme.spacing(10),
      },

Reply:
Also, you can add:
```
marginLeft: theme.spacing(0),
marginRight: theme.spacing(0),
```